oracle/README.md

```diff
@@ -7,13 +7,13 @@ Tests are ran against our own test vectors in `tests/test_vectors`.
 You can re-generate the test vectors by using:
 
 ```text
-usage: cargo run --all-features -- [Hex|B10] [legacy|kimchi] <OUTPUT_FILE>
+usage: cargo run -p export_test_vectors -- [Hex|B10] [legacy|kimchi] <OUTPUT_FILE>
 ```
 
 Examples
 
 ```text
-cargo run --all-features -- B10 legacy -
-cargo run --all-features -- b10 legacy legacy.json
-cargo run --all-features -- hex kimchi kimchi.json
+cargo run -p export_test_vectors -- B10 legacy -
+cargo run -p export_test_vectors -- b10 legacy legacy.json
+cargo run -p export_test_vectors -- hex kimchi kimchi.json
```

Comment:
looks like it works without the `-p` option as well (probably because there's only one binary)

Reply:
i tried running in the oracle folder directly and its complaining when i didnt have `-p` .  however it works well without `-p` when i run in the `export_test_vectors` folder. wondering if you are seeing the same too

Reply:
oh wait I'm stupid, you're right!